src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkWindows.cs

```diff
@@ -230,6 +230,10 @@ protected override IEnumerable<string> GetAllAvailableAndroidNdks ()
 				if (CheckRegistryKeyForExecutable (root, regKey, MDREG_ANDROID_NDK, wow, ".", NdkStack))
 					yield return RegistryEx.GetValueString (root, regKey, MDREG_ANDROID_NDK, wow) ?? "";
 
+			foreach (string dir in GetSdkFromEnvironmentVariables ()) {
```

Comment:
This is "dead code", as `GetAllAvailableAndroidNdks()` is only called from `AndroidSdkBase.GetValidNdkPath()`.

Reply:
This is the code I've just added...

Reply:
Never mind. sorry!

I thought `FindBestNDK()` called `GetSdkFromEnvironmentVariables()` as well, and was mistaken.